### ISSUE TITLE
Migrate to Rails

### ISSUE THREAD

**Issue by ivanoblomov** [Spark-Bio/covid-storepoint#37]
Given:
1. Maintaining our own datastore will make reconciling location data from multiple sources easier.
2. Allowing users to correct/update location info would be a great value-add.

Migrating to Rails makes sense.

Extracted from https://github.com/Spark-Bio/covid-storepoint/issues/10#issuecomment-614428722.

**Comment by ivanoblomov** [comment]
@neilberkman happy to take this on, but just want to coordinate a few things:
1. timing: when's a good time to start (given the issues/PRs in progress and new team members)?
2. best approach: new repo or running the Rails generator in the existing repo? Haven't converted plain-old-ruby code to Rails before, so was wondering best practice here in case you have.

**Comment by ivanoblomov** [comment]
Based on discussion with @neilberkman re: the large volunteer effort GIS runs to maintain the data, it doesn't make sense to duplicate their effort with our own admin interface.

TL;DR: We're gonna table an app server for now and only add persistence later, if we need it.
